### DISCUSSION TITLE
How to catch 429 Too Many Requests: retry after X time errors

### DISCUSSION THREAD

**Discussion by espipj** [telegraf/telegraf#1805] [Q&A]
And avoid the bot crashing by retrying the request after X time

I'm getting this error:
```json
response: {
    ok: false,
    error_code: 429,
    description: 'Too Many Requests: retry after 43',
    parameters: { retry_after: 43 }
  },
  on: {
    method: 'sendMediaGroup',
    payload: { chat_id: '-10017', media: [Array] }
  }
}

```
May be related to #1563

**Comment by MKRhere** [comment]
No, this is not related to #1563. Are you trying to broadcast messages to a large number of users? Consider reading: [Broadcasting | Telegraf Docs](https://github.com/feathers-studio/telegraf-docs/blob/4feffcd893a86ed1fcc761279d17328beb4a2f21/guide/%3F%3F%3F-broadcasting.md)
